openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -0,0 +1,256 @@
+"""
+Tests for course_overviews app.
+"""
+import datetime
+import ddt
+import itertools
+import pytz
+import math
+
+from django.utils import timezone
+
+from lms.djangoapps.certificates.api import get_active_web_certificate
+from lms.djangoapps.courseware.courses import course_image_url
+from xmodule.course_metadata_utils import DEFAULT_START_DATE
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, check_mongo_calls, check_mongo_calls_range
+
+from .models import CourseOverview
+
+
+@ddt.ddt
+class CourseOverviewTestCase(ModuleStoreTestCase):
+    """
+    Tests for CourseOverviewDescriptor model.
+    """
+
+    TODAY = timezone.now()
+    LAST_MONTH = TODAY + datetime.timedelta(days=30)
```

Comment:
LAST_MONTH is only used in one place where the end date is sooner than the start date. I wonder if this is allowed in studio?